pkg/cmd/target/set.go

```diff
@@ -116,7 +116,7 @@ var TargetSetCmd = &cobra.Command{
 		var selectedTarget *target_view.TargetView
 
 		if !isNewProvider || len(filteredTargets) > 0 {
-			selectedTarget, err = target.GetTargetFromPrompt(filteredTargets, activeProfile.Name, nil, true)
+			selectedTarget, err = target.GetTargetFromPrompt(filteredTargets, activeProfile.Name, nil, true, "Set")
```

Comment:
We should add `and set as default` to the command output when a new target is added and set as default.